Cloud Application Manager/Core Concepts/providers.md

```diff
@@ -1,29 +1,54 @@
 {{{
 "title": "Providers",
-"date": "06-16-2017",
+"date": "10-17-2018",
 "author": "",
+"keywords": "CAM, ALM, Providers, CenturyLink Cloud, AWS, Azure, Google Cloud, CloudStack, Dimension Data, OpenStack, Rackspace, SoftLayer, VCloud, VMWare",
 "attachments": [],
 "contentIsHTML": false
 }}}
 
+### Providers <!-- omit in toc -->
+
+**In this article:**
+
+* [Overview](#overview)
+* [Audience](#audience)
+* [Prerequisites](#prerequisites)
+* [Providers](#providers)
+* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+
+### Overview
+
+Providers is part of the basics of Cloud Application Manager. This article is meant to assist Cloud Application Manager customers who are starting using the plataform.
+
+### Audience
+
+All Cloud Application Manager customers.
+
+### Prerequisites
+
+An active Cloud Application Manager account.
+
 ### Providers
 
 A provider is a public or private cloud account you register in Cloud Application Manager. Cloud Application Manager interfaces with the provider account’s API to provision and orchestrate deployments automatically. Before you can deploy workloads from Cloud Application Manager, you need to register a provider first. Cloud Application Manager integrates with many popular clouds. Here’s a full list of clouds Cloud Application Manager supports and their availability by Edition:
 
```

Comment:
I suggest to add a new section called "Supported provider types" and include there the last sentence of the previous paragraph and the table below. Please also add it to TOC.

Reply:
Done in a previous update.